src/WebJobs.Extensions.NotificationHubs/Services/INotificationHubClientService.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.NotificationHubs;
+
+namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
+{
+    /// <summary>
+    ///   Defines an interface for NotificationHubClient sendNotification
+    /// </summary>
+    public interface INotificationHubClientService
```

Comment:
Why is this public? Looks like it's only for internal use, not customers (unless I missed it)